### ISSUE TITLE
Where did official repo go

### ISSUE THREAD

**Issue by hiroki-chen** [YurkoWasHere/tdx-tools#1]
Hi Yurko, thanks for the mirror repo, but I'm curious about the whereabouts of the official repo. I followed NVIDIA's deployment and the `tdx-tools` repo simply didn't exist 😂. Do you have any idea?

Thanks!

**Comment by YurkoWasHere** [comment]
I have no idea where it went. I was doing the same thing as you, setting up this stack on on a new machine and could not find this repo.  Luckily i had a clone sitting on another machine an was able to upload it to github.

I know the NVidia documentation is targeting the an old stack. Newer TDX work is being done on https://github.com/canonical/tdx however i have not yet been successful in getting the TDX+NVIDIA to work with the new TDX stacks.

**Comment by hiroki-chen** [comment]
I heard from NVIDIA that the repo you mentioned (https://github.com/canonical/tdx) does not include the passthrough support so failure to make H100+TDX work is expected.

**Comment by YurkoWasHere** [comment]
Yes its a bit of a mess,  As i understand it there is a branch that supposed to work with the H100 and the newer kernels but im having trouble booting the TDX in CC mode even without the H100.

https://github.com/canonical/tdx/issues/185
